NibDesignable/NibDesignableTests/NibDesignableTests.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
```

Comment:
This file should either be deleted and the test target as well or the tests from the demo project should be move here.